BLSQ/iaso
# IA-1897: remove search filter component

SearchFilter commponent was not used anymore anywhere but here

Related JIRA tickets : [IA-1897](https://bluesquare.atlassian.net/browse/IA-1897?atlOrigin=eyJpIjoiMjQ5Nzc5MjIzMzM1NGJlZjk3ZmMzMGQ0NDczN2E4ZTkiLCJwIjoiaiJ9)

## Self proofreading checklist


## Changes

- Remove the `SearchFilter` obselete component
-  Replaced by `InputComponent` type="search"
- Replaced the SearchButton by FilterButton
-  Add a `setFiltersUpdated` prop to OrgunitsFilters to keep the same logic elsewhere in the app disabling the search before any change

## How to test

- Go to Orgunits list
- Try the Search InputComponent
It should work exactly the same as before but now the FilterButton is disabled until a filter input has  a value

## Print screen / video

https://user-images.githubusercontent.com/25134301/220413813-e1843f9f-5097-4ed2-a6b5-70f2d0ea34e5.mov


[IA-1897]: https://bluesquare.atlassian.net/browse/IA-1897?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
